test/integration/deploy_int_test.go

```diff
@@ -158,7 +158,6 @@ func (suite *DeployIntegrationTestSuite) TestDeployPython() {
 	defer ts.Close()
 
 	ts.SetupRCFile()
-	suite.T().Setenv("ACTIVESTATE_HOME", ts.Dirs.HomeDir)
```

Comment:
I don't understand why these are being removed in this file. I feel like they are important.

Reply:
They are which is why this is set as part of the environment sandboxing.

Reply:
Ohhh, I see. I wasn't aware it was done elsewhere.